### ISSUE TITLE
Bug Report: `CAN_OPEN_LINKS` isn't defined anymore (Folder opening is broken)

### ISSUE THREAD

**Issue by NotHyper-474** [FunkinCrew/Funkin#3340]
### Issue Checklist

- [X] I have properly named the issue
- [X] I have checked the issues/discussions pages to see if the issue has been previously reported

### What platform are you using?

Itch.io (Downloadable Build) - Windows

### If you are playing on a browser, which one are you using?

None

### Version

0.5.0

### Context (Provide images, videos, etc.)

This is likely related to the replacement of Project.xml with project.hxp since that's where this define was originally implemented.
Since this define was used by both folder and URL opening this causes a crash when clicking "Take Me There" on the chart editor's backup popup or other buttons that open a folder.

### Steps to reproduce (or crash logs, errors, etc.)
* Go to Chart Editor
* Click on Help > Open Backup Folder
* Crash :)

```
Cannot open URLs on this platform.
  in funkin/util/WindowUtil.hx#52
  in funkin/ui/debug/charting/ChartEditorState.hx#3243
  in funkin/ui/debug/charting/handlers/ChartEditorDialogHandler.hx#198
  in haxe/ui/util/EventMap.hx#90
  in haxe/ui/backend/ComponentBase.hx#441
  in haxe/ui/backend/flixel/MouseHelper.hx#201
  in haxe/ui/backend/flixel/MouseHelper.hx#191
  in haxe/ui/backend/flixel/MouseHelper.hx#173
  in haxe/ui/backend/flixel/MouseHelper.hx#103
```

**Comment by NotHyper-474** [comment]
Apparently this is also why the merch button isn't showing up anymore

**Comment by EliteMasterEric** [comment]
Resolved by #3342
